lib/Importer/Caffe2ModelLoader.cpp

```diff
@@ -333,7 +333,8 @@ llvm::Error Caffe2ModelLoader::loadConv(const caffe2::OperatorDef &op,
   // Caffe2 "Conv" op always stores the weight as CKRS.
   Tensor wT;
   w->getPayload().transpose(&wT, NCHW2NHWC);
-  w = G_.getParent()->createConstant(w->getName(), std::move(wT));
+  w = G_.getParent()->createConstant(w->getName(), std::move(wT),
+                                     GLOW_DEFAULT_4D_LAYOUT);
```

Comment:
Please use a concrete string here, because any change to the definition of GLOW_DEFAULT_4D_LAYOUT would break this code.